microcosm_pubsub/daemon.py

```diff
@@ -12,7 +12,7 @@ class ConsumerDaemon(Daemon):
 
     def make_arg_parser(self):
         parser = super(ConsumerDaemon, self).make_arg_parser()
-        parser.add_argument("--sqs-queue-url", required=True)
+        parser.add_argument("--sqs-queue-url")
```

Comment:
I'm specifically changing my daemon to have a reasonable default for the queue URL; it makes local development configuration a lot simpler.